Revise Getting Started section (#1513)
getsentry/sentry-docs: src/collections/_documentation/error-reporting/getting-started-verify/browser.md

What better way to send your first event than to break your application!

> One way to break your JavaScript application is to call an undefined function:

Reviewer: Maybe reword to "One way to do this (that?) is..." to not repeat "break your" twice?

Author: This repetition was already there for JS Browser, Cordova, JS Node, Electron. Now it just got obvious without rendering the docs :)

**CURRENT** https://docs.sentry.io/error-reporting/quickstart/?platform=browser#capturing-your-first-event
![image](https://user-images.githubusercontent.com/88819/75336176-f3f6f280-588a-11ea-95eb-1999549316ad.png)

Not too bad, is it? I'll reword it since we're already touching this section.

> What better way to send your first event than to break your application!

Reviewer 2: ```suggestion
One way to verify your setup is by intentionally sending an event that breaks your application. 
```

> One way to do it is to call an undefined function:

Reviewer 2: ```suggestion
Calling an undefined function will trigger a break:```

Author: "trigger a break" sounds really strange. "break" in JS is a keyword unrelated to exceptions.

How about "throw an exception", instead?

```suggestion
Calling an undefined function will throw an exception:
```

Reviewer 2: "throw an exception" is great!
